src/common/mapindex.c

```diff
@@ -134,29 +132,44 @@ void mapindex_init(void) {
 	int last_index = -1;
 	int index;
 	char map_name[MAP_NAME_LENGTH];
+	char* mapindex_cfgfile[80] = {
+		"db/map_index.txt",
+		"db/import/map_index.txt"
```

Comment:
As you can see in b5e30e0 it was hardcoded before too. I will adjust this later today. ;)